platform/node/README.md

```diff
@@ -14,7 +15,7 @@ By default, installs binaries. On these platforms no additional dependencies are
 Run:
 
 ```
-npm install @mapbox/mapbox-gl-native
+npm install @maplibre/maplibre-gl-native
```

Comment:
this is likely to confuse users until this package is actually available.  Suggest adding a big warning block at the top of this file to indicate that this is planned (I assume?) but does not yet exist.

Reply:
Good call 👍